require.js

```diff
@@ -1228,15 +1292,16 @@
 
     Require.makeCompiler = function (config) {
         return function (module) {
-            return new Promise(function (resolve, reject) {
-                return Require.MetaCompiler(module).then(function () {
-                    if (typeof module.exports === "object") {
-                        resolve(module);
-                    } else {
-                        resolve(syncCompilerChain(config)(module));
-                    }
-                });
-            });
+            // return new Promise(function (resolve, reject) {
+            //     return Require.MetaCompiler(module).then(function () {
+            //         if (typeof module.exports === "object") {
+            //             resolve(module);
+            //         } else {
+            //             resolve(syncCompilerChain(config)(module));
+            //         }
+            //     });
+            // });
+            return Promise.resolve(syncCompilerChain(config)(module));
```

Comment:
If this is all `makeCompiler` does now, maybe we can elide this method?

Reply:
no because makeCompiler is an API on the config and can be a custom one provided from the outside, this is the default

Reply:
Ok